### PR TITLE
Subnet ip usage v1.1

### PR DESCRIPTION
**Pull request by rsagasthya**
## Changes:
1. Workbook Look and Feel changes: Ref: https://microsoft-my.sharepoint.com/:o:/p/ragasthya/Eq3O1Khhj2RDtlYMhSVqvLEBTyAQMpYN6DZNW2L-Fyl44Q
2. Included a new Tab to display all IP Usage Metrics.

## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/AKS/Subnet IP Usage/Subnet IP Usage.workbook [hunk 1]
```diff
@@ -483,18 +491,21 @@
             "type": 3,
             "content": {
               "version": "KqlItem/1.0",
-              "query": "InsightsMetrics\n| where TimeGenerated < {timeRange:end} and TimeGenerated >= {timeRange:start}\n| where Name == \"cx_ipam_pod_allocated_ips\"\n| where Namespace == \"prometheus\"\n| extend jsonprop = parse_json(Tags)\n| extend HostName = jsonprop.hostName\n{hostwhereclause}\n| extend podnet_arm_id = jsonprop.podnet_arm_id\n| project TimeGenerated, Val, podnet_arm_id, HostName\n| summarize count = count(Val) by bin(TimeGenerated, {timeRange:grain}), tostring(HostName)\n\n",
+              "query": "InsightsMetrics\n| where TimeGenerated < {timeRange:end} and TimeGenerated >= {timeRange:start}\n| where Name == \"cx_ipam_pod_allocated_ips\"\n| where Namespace == \"prometheus\"\n| extend jsonprop = parse_json(Tags)\n| extend HostName = jsonprop.hostName\n{hostwhereclause}\n| project TimeGenerated, Val, HostName\n| summarize count = count(Val) by bin(TimeGenerated, {timeRange:grain}), tostring(HostName), Val\n\n",
```
Comment (vishiy) [inline Workbooks/AKS/Subnet IP Usage/Subnet IP Usage.workbook:494]: why are we summarizing by 'Val' ?

File: Workbooks/AKS/Subnet IP Usage/Subnet IP Usage.workbook [hunk 2]
```diff
@@ -170,21 +170,25 @@
...
+                  "Val"
+                ],
                 "group": "HostName",
-                "createOtherGroup": null
+                "createOtherGroup": null,
+                "showLegend": true
               }
             },
-            "name": "query - 1"
+            "name": "IPs allocated from Subnet"
```
Comment (vishiy) [inline Workbooks/AKS/Subnet IP Usage/Subnet IP Usage.workbook:191]: can you also please share screen shot for query changes ? (before & after) ?

Comment (rsagasthya) [reply Workbooks/AKS/Subnet IP Usage/Subnet IP Usage.workbook:436]: Included in a OneNote. Shared it in the main commit comment.

File: Workbooks/AKS/Subnet IP Usage/Subnet IP Usage.workbook (hunk 1 above)
Comment (rsagasthya) [reply Workbooks/AKS/Subnet IP Usage/Subnet IP Usage.workbook]: We were summarizing by count(Val). The metric values were off. Removed that and using only Val.

File: Workbooks/AKS/Subnet IP Usage/Subnet IP Usage.workbook (hunk 2 above)
Comment (vishiy) [reply Workbooks/AKS/Subnet IP Usage/Subnet IP Usage.workbook:436]: Thanks. i get access denied for that one note.

**Review by vishiy — APPROVED** [review]
approve after seeing the sample charts.

**Pull request merged**